src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  let basePrice = 0;
```

Comment:
Prefer `const` over `let` where possible, to avoid unintentional changes. Since `basePrice` is assigned only once, it should be declared as a `const`.